package.json

```diff
@@ -6,32 +6,35 @@
   "author": "The KeystoneJS Development Team",
   "license": "MIT",
   "scripts": {
-    "coverage": "DISABLE_LOGGING=true NODE_ENV=test jest --maxWorkers=1 --coverage",
+    "demo-projects": "yarn --cwd ./demo-projects",
+    "test-projects": "yarn --cwd ./test-projects",
+    "website": "yarn --cwd ./website",
+    "coverage": "cross-env DISABLE_LOGGING=true NODE_ENV=test jest --maxWorkers=1 --coverage",
     "cypress:run": "status=0; bolt w @keystone-alpha/cypress-project-basic run cypress:run || status=$?; bolt w @keystone-alpha/cypress-project-login run cypress:run || status=$?; bolt w @keystone-alpha/cypress-project-twitter-login run cypress:run || status=$?; bolt w @keystone-alpha/cypress-project-access-control run cypress:run || status=$?; exit $status",
     "format:file": "prettier --write",
     "format": "prettier --write \"**/*.js{,on}\" \"**/*.md\" && remark . --output",
     "fresh": "yarn clean && bolt",
-    "clean": "bolt ws exec -- rm -rf node_modules && rm -rf node_modules && rm -rf website/.cache",
+    "clean": "bolt ws exec -- rimraf node_modules website/.cache",
     "lint:eslint": "eslint .",
     "lint:prettier": "prettier --list-different \"**/*.js\"",
     "lint:flow": "flow check",
     "lint:markdown": "remark . --frail --quiet",
     "lint": "yarn lint:prettier && yarn lint:eslint && yarn lint:flow && yarn lint:markdown",
     "start": "yarn start:demo",
-    "start:demo": "/bin/sh -c 'cd demo-projects/${1:-$0} && yarn start' todo",
-    "start:test": "/bin/sh -c 'cd test-projects/${1:-$0} && yarn start' basic",
-    "website:start": "cd website && yarn start",
-    "website:build": "cd website && yarn build",
+    "start:demo": "yarn demo-projects start",
+    "start:test": "yarn test-projects start",
+    "website:build": "yarn website build",
+    "website:start": "yarn website start",
```

Comment:
We can ditch all these as long as `yarn website start`/`yarn demo blog start` works from the root directory.

Reply:
I think will need to update (serveral) `readme.md` too

Reply:
Yeah, a quick grep for `(website|start):.*` should find all the uses